src/vs/code/electron-browser/sharedProcess/sharedProcessMain.ts

```diff
@@ -123,6 +121,7 @@ function main(server: Server, initData: ISharedProcessInitData, configuration: I
 		} else {
 			services.set(ITelemetryService, NullTelemetryService);
 		}
+		server.registerChannel('telemetryAppender', new TelemetryAppenderChannel(appInsightsAppender));
```

Comment:
What happens if something tries to send to this channel and it's not registered?